crates/uv/tests/it/pip_install.rs

```diff
@@ -2174,6 +2174,37 @@ fn install_github_artifact_private_https_multiple_pat() {
     context.assert_installed("uv_private_pypackage", "0.1.0");
 }
 
+/// Install a package from a private GitHub repository using interactive authentication
+/// Regression test for <https://github.com/astral-sh/uv/issues/5107>
+#[test]
+#[cfg(feature = "git")]
+fn install_git_private_https_interactive() {
+    let context = TestContext::new("3.8");
+    let token = decode_token(common::READ_ONLY_GITHUB_TOKEN);
```

Comment:
Do we need the token here if we're not going to use it?

Reply:
Well the original idea for the test was to test that login via CLI works, but now that this tests a failure instead I can indeed remove it.